css/sections/bento.css

```
@@ -1,1175 +1,1185 @@
-	overflow: hidden;
-	background-color: var(--paper-background);
-	position: relative;
-	justify-content: space-between;
-	flex-direction: column;
-	overflow: hidden;
-	transition: box-shadow 0.2s cubic-bezier(0.645, 0.045, 0.355, 1);
+  grid-column: span 2;
+  min-height: 280px;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  overflow: hidden;
```

Stylelint (17.2.0)

[error] Unexpected duplicate "overflow" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)